api/internal/repository/nft_token.go

```diff
@@ -75,6 +75,14 @@ func (n *nftTokenDbImpl) GetNfts(collectionAddresses []string, walletAddress str
 		query = query.Where("owner_address = ?", walletAddress)
 	}
 
+	if offset != nil {
```

Comment:
any sanity checking on the offset?

Reply:
Added validation to check if offset is > 0 in the handler. For offset that is larger than the total db record, the query will return empty array.